addon/components/dynamic-form.js

```diff
@@ -21,12 +22,45 @@ const DynamicForm = Ember.Component.extend({
 
   didReceiveAttrs() {
     let schemaObj = this._initSchema(this.get('schema'));
-    const schemaWithActions = this._addActions(schemaObj);
+    const schemaWithPostRender = this._buildPostRender(schemaObj);
+    const schemaWithActions = this._addActions(schemaWithPostRender);
     const filteredSchema = this._processFilters(schemaWithActions);
     const mappedSchema = this._replaceKeywordsWithFunctions(filteredSchema);
     this.set('renderSchema', mappedSchema);
   },
 
+  _buildPostRender(schemaObj) {
```

Comment:
This new function is the meat of the feature.  The logic here aggregates the caller-provided postRender, the postRender in the schema, and the changeAction postRender, and calls them as one postRender function.

For the changeAction, we iterate all fields and add a change handlers.